app/registries/foreman/access_permissions.rb

```diff
@@ -319,8 +319,8 @@
                                     :puppetclasses => pc_ajax_actions,
                                     :subnets => subnets_ajax_actions,
                                     :interfaces => [:new, :random_name],
-                                     :"api/v2/hosts" => [:create],
-                                     :"api/v2/interfaces" => [:create],
+                                     :"api/v2/hosts" => [:create, :clone],
+                                     :"api/v2/interfaces" => [:create, :clone],
```

Comment:
I do not see a `clone` action in that controller...

Reply:
Removed